src/edit_collector.h

```diff
@@ -104,6 +105,17 @@ class EditCollector {
     return Status::Corruption("No next file number in manifest file");
   }
 
+  void Dump(bool with_keys) const {
+    if (has_next_file_number_) {
+      fprintf(stdout, "next_file_number: %" PRIu64 "\n", next_file_number_);
+      for (auto& cf : column_families_) {
+        fprintf(stdout, "column family: %" PRIu32 "\n", cf.first);
+        cf.second.Dump(with_keys);
+        fprintf(stdout, "\n");
```

Comment:
Is this necessary?

Reply:
Just an additional empty line between CFs.